lib/core/test/unit/Cardano/Wallet/DBSpec.hs

```diff
@@ -354,7 +355,11 @@ instance Arbitrary GenTxHistory where
     -- Ensure unique transaction IDs within a given batch of transactions to add
     -- to the history.
     arbitrary = GenTxHistory . Map.fromList <$> do
-        txs <- arbitrary
+        -- NOTE
+        -- We discard pending transaction from any 'GenTxHistory since,
+        -- inserting a pending transaction actually has an effect on the
+        -- checkpoint's pending transactions of a same wallet.
```

Comment:
Perhaps:
```hs
        -- We discard pending transactions from any 'GenTxHistory', since
        -- inserting a pending transaction actually has an effect on the
        -- checkpoint's pending transactions of the same wallet.
```